console/console-init/ui/src/modules/iot-device/components/AddDeviceWithJson/AddDeviceWithJson.tsx

```diff
@@ -246,11 +299,10 @@ const AddDeviceWithJson: React.FunctionComponent<IAddDeviceWithJsonProps> = ({
   const getEditor = () => {
     return (
       <JsonEditor
-        value={deviceJsonString}
+        value={device}
```

Comment:
need to use deviceJsonString instead of device.
device state will use for final data for save and validate.